community-membership.md

```diff
@@ -1,77 +1,198 @@
 # Community membership
 
-New community members:
-
-- [**New Contributor**](https://github.com/kubernetes/contrib/issues/1090): a
-  couple of PRs; should be welcomed to the community, helped with PR workflow, and
-  directed to relevant documentation
-- **Active Contributor**: at least 3 merged and/or assigned PRs (which could include documentation
-  contributions as well as code), including one in the past month; we have
-  [expectations](contributors/devel/community-expectations.md)
-  that frequent contributors will assist in our code-review process and with project
-  maintenance
-  
-Established community members:
+This doc outlines the various responsibilities of contributor roles in Kubernetes.  The Kubernetes
+project is subdivided into various sub-areas.  Responsibilities for many contributor roles are
+scoped by these sub-areas.
+
+| Role | Responsibilities | Requirements | Defined by |
+| -----| ---------------- | ------------ | -------|
+| member | Active contributor in the community | Sponsored by 2 members.  Already active | Outside collaborator with read access. |
+| reviewer | Review contribution from other members | History of review and authorship in an area | Reviewer entry in an OWNERS file |
+| approver | Approve accepting contributions | Highly experienced and active reviewer + contributor to an area | Approver entry in an OWNERS file|
+| owner | Set priorities and approve proposals | Primary or major contributor and maintainer of area | Approver entry in a top-level area OWNERS file |
```

Comment:
How would an observer figure out if someone is an "approver" or "owner" if both are "Approver" entry in the OWNERS file?